nerdlandbot/commands/reminder.py

```diff
@@ -0,0 +1,96 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+from datetime import datetime
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.constants import *
+
+
+class Reminder(commands.Cog, name="reminder"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="reminder", aliases=["remind_me"], brief="reminder_brief", usage="reminder_usage", help="reminder_help")
+    async def reminder(self, ctx: commands.Context, time: typing.Optional[int] = None, *, message: typing.Optional[str] = None):
```

Comment:
Recommend to put this as string then try to convert to int, if convert fails then (try catch) post message about time format being wrong and needing an integer for minutes.

Reply:
Changed!